src/InterpreterGenerator.h

```diff
@@ -415,6 +416,14 @@ class NodeGenerator : public RamVisitor<std::unique_ptr<InterpreterNode>> {
         return std::make_unique<InterpreterNode>(I_Exit, &exit, std::move(children));
     }
 
+    NodePtr visitCall(const RamCall& call) override {
+        std::vector<size_t> data;
```

Comment:
What is `data`? Is there a more descriptive name? I can't work out what it's for without digging through all the methods that are called.

Reply:
This is unformatted data in the InterpreterNode and has no specific meaning. Depending on node, parameters are stored in this array